website_field_autocomplete/controllers/main.py

```diff
@@ -6,6 +6,8 @@
 from odoo import http
 from odoo.http import request
 from odoo.addons.website.controllers.main import Website
+import logging
```

Comment:
Why the changes in this file?

Reply:
Sorry, that shouldn't be there 👍 
Fixed